users/introducer.rst

```diff
@@ -0,0 +1,14 @@
+.. _introducer:
+
+Introducer Configuration
+========================
+
+The introducer feature allows a degree of automation for adding unconnected devices to a cluster. When two devices connect they exchange a list of mutually shared folders and the devices connected to those shares. If a device has a remote device configured as an introducer it will then act on this information, adding unknown devices listed by the introducer and connecting them to the currently configured folders as per the introducer's settings. This is done on a per-folder basis and only devices which are in common with mutual shares are included. No additional shares are imported, nor are devices which are not in common between the introducer and device's shared folder list.
```

Comment:
A degree of automation might be too hard to google translate and understand for non native speakers.

"If device has remot device" maybe it would be easier to explain using A B C concrete examples with multiple folders making it clear what happens.